src/lib/builtins/array.rs

```diff
@@ -326,14 +326,14 @@
~            "missing callback when calling function Array.prototype.every".to_string(),
~        ));
~    }
~    let callback = &args[0];
~    let this_arg = if args.len() > 1 {
~        args[1].clone()
~    } else {
```

Comment:
Looks like `Array.prototype.length` shouldn't have a getter according to the spec